packages/itwin/measure-tools/src/widgets/PopupToolbar.tsx

```diff
@@ -18,14 +18,13 @@ export interface PopupToolbarProps extends ToolbarProps {
  * handler.
  */
 export class PopupToolbar extends React.PureComponent<PopupToolbarProps> {
-  private _toolbar: Toolbar;
+  private _toolbar: JSX.Element;
```

Comment:
is this right?

Reply:
with the change in what toolbar we import, this change was neccesary